oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/components/filepreview.js

```diff
@@ -0,0 +1,18 @@
+import $ from "jquery";
+
+$("#preview-modal").modal();
+const iframe = document.querySelector(".fileIframe");
+
+function openModal(event) {
+  const fileTitle = event.target.dataset.title;
```

Comment:
Whole preview url should come from data attribute (& from file links) and should not be concatenated like this

Reply:
There is no added preview url https://github.com/Narodni-repozitar/nr-docs/blob/b3e4ad113ec8bd79648c6e0b254e1993c75598c7/documents/services/files/config.py#L41